pkg/v1/providers/ytt/vendir/kapp-controller/_ytt_lib/addons/packages/kapp-controller/0.30.0/bundle/config/overlays/update-deployment.yaml

```diff
@@ -29,10 +29,42 @@ spec:
           #@overlay/match by="name"
           - name: KAPPCTRL_API_PORT
             value: #@ str(values.kappController.deployment.apiPort)
+        #@ if values.kappController.deployment.hostNetwork:
```

Comment:
This overlay should be moved to upstream package in TCE as it applicable there as well.

Reply:
Yes, it's vendired down from TCE